deployment/provision_vagrant_vm.sh

```diff
@@ -34,3 +41,8 @@ git submodule update --init
 sudo -H -u vagrant python3 manage.py migrate --noinput
 sudo -H -u vagrant python3 manage.py collectstatic --noinput
 sudo -H -u vagrant python3 manage.py compilemessages
+
+
+# setup static files
+cd /vagrant/_1327/static
+sudo -H -u vagrant yarn
```

Comment:
```suggestion
sudo -H -u vagrant yarn --no-bin-links
```
this should fix the problem using a vagrant vm on windows